src/main/java/com/nirima/jenkins/plugins/docker/utils/PortUtils.java

```diff
@@ -172,16 +172,16 @@ public boolean execute() throws InterruptedException {
         }
 
         private boolean executeOnce(final int thisTryNumber, final int totalTriesIntended) {
-            final Connection connection = new Connection(parent.host, parent.port);
+            final Connection sshConnection = new Connection(parent.host, parent.port);
```

Comment:
I renamed the variable to make it clear that this was a SSH connection not merely a TCP connection that was being made.